.circleci/config.yml

```diff
@@ -37,7 +42,10 @@ jobs:
       - checkout
       - run: dnf install -y pkg-config ragel gcc gcc-c++ automake autoconf libtool make which glib2-devel freetype-devel cairo-devel libicu-devel gobject-introspection-devel graphite2-devel redhat-rpm-config python || true
       - run: NOCONFIGURE=1 ./autogen.sh --with-freetype --with-glib --with-gobject --with-cairo --with-icu --with-graphite2
-      - run: mkdir build && cd build && ../configure && make && (make check || (.ci/collect-logs.sh && false))
+      - run: mkdir build && cd build
+      - run: ../configure
```

Comment:
Working directory won't be preserved thus better to keep this on one line.

Reply:
Didn’t know that. Fixed.